samples/nrf9160/sb_fota_client/prj.conf

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# General config
+CONFIG_ASSERT=y
+CONFIG_REBOOT=y
+CONFIG_BASE64=y
+CONFIG_NEWLIB_LIBC=y
+CONFIG_FPU=y
+
+# Heap pool
+CONFIG_HEAP_MEM_POOL_SIZE=16384
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=8192
+CONFIG_MAIN_STACK_SIZE=2048
+
+# Modem library
+CONFIG_NRF_MODEM_LIB=y
+CONFIG_NRF_MODEM_LIB_SYS_INIT=y
+CONFIG_MODEM_INFO=y
+
+# Network
+CONFIG_NETWORKING=y
+CONFIG_NET_NATIVE=n
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+
+# Shell
+CONFIG_SHELL=y
+CONFIG_SHELL_WILDCARD=n
+
+# Network
+CONFIG_NETWORKING=y
+CONFIG_NET_NATIVE=n
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+
+# LTE link control
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_AUTO_INIT_AND_CONNECT=y
+CONFIG_LTE_EDRX_REQ_VALUE_LTE_M="0101"
+CONFIG_LTE_EDRX_REQ_VALUE_NBIOT="0101"
+CONFIG_LTE_PSM_REQ_RAT="00100001"
+CONFIG_LTE_PSM_REQ_RPTAU="00000001"
```

Comment:
I think it would be good to add a comment for each value (what that means in seconds/minutes).